test/test_utils.sh

```diff
@@ -174,14 +174,16 @@ helm_install_fission() {
     echo "Deleting old releases"
     helm list -q|xargs -I@ bash -c "helm_uninstall_fission @"
 
+    # deleting ns does take a while after command is issued.
+    sleep 45
+
     echo "Installing fission"
     helm install		\
 	 --wait			\
-	 --timeout 600	        \
+	 --timeout 540	        \
 	 --name $id		\
 	 --set $helmVars	\
 	 --namespace $ns        \
-	 --debug                \
```

Comment:
isn't --debug useful?

Reply:
I just felt the info it dumped was repeated later as part of the `dump objects` and `dump logs` functions. Also, it used up 801 lines in the log. I can add it back if its needed.

Reply:
Makse sense, no need to add it back then.